dmriprep/utils/vectors.py

```diff
@@ -375,3 +375,58 @@ def bvecs2ras(affine, bvecs, norm=True, bvec_norm_epsilon=0.2):
         rotated_bvecs[~b0s] /= norms_bvecs[~b0s, np.newaxis]
         rotated_bvecs[b0s] = np.zeros(3)
     return rotated_bvecs
+
+
+def _nonoverlapping_qspace_samples(
```

Comment:
Could we make this a publicly available function? Could be useful in different contexts